package.json

```diff
@@ -7,11 +7,11 @@
     "test": "NODE_ENV=test sequelize db:migrate:undo:all && npm run db:migrate:test && npm run db:migrate:seed:test && NODE_ENV=test nyc mocha --require @babel/register --timeout 30000 src/tests/index.js",
     "cover": "npm test && nyc report --reporter=text-lcov | coveralls",
     "coveralls": "cat coverage/lcov.info | coveralls",
-    "start": "npm run db:migrate && babel-node src/index.js",
-    "dev": "nodemon src/index.js --exec babel-node --presets @babel/preset-env",
+    "start": "concurrently \"npm run db:migrate && npm run db:seed\" \"babel-node src/index.js\"",
+    "dev": "concurrently \"npm run db:seed\"  \"nodemon src/index.js --exec babel-node --presets @babel/preset-env\"",
     "scss": "npm-sass templates/sass/style.scss templates/css/style.css",
     "db:migrate": "./node_modules/.bin/sequelize db:migrate",
-    "db:migrate:seed": "./node_modules/.bin/sequelize db:seed:all",
+    "db:seed": "./node_modules/.bin/sequelize db:seed --seed 20190605075225-permissions",
```

Comment:
why is the seeding done form one file?

Reply:
the only thing that we need to seed for now is permissions, but we can still add other seed when they are needed

Reply:
let's just have it as seed all and remove other seeder files.